src/core/FileUpload/FileUploadTemplate.tsx

```diff
@@ -57,6 +63,7 @@ export const FileUploadTemplate = (props: FileUploadTemplateProps) => {
         id='file-input'
```

Comment:
I just saw that we are setting `id`, this can cause errors if 2 `FileUploadTemplate` components are present. We need to have a prop for this.

Reply:
`inputProps` would fix this too 😄

Reply:
I will remove `id` and `for` after https://github.com/iTwin/iTwinUI/issues/146 is available.

But I think we would still need to support adding `id`. The user might have a use case for it 🤔